control/control_performance_analysis/package.xml

```diff
@@ -10,12 +10,14 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
-  <depend>autoware_control_msgs</depend>
+  <depend>autoware_auto_control_msgs</depend>
+  <depend>autoware_auto_planning_msgs</depend>
+  <depend>autoware_auto_vehicle_msgs</depend>
   <depend>autoware_planning_msgs</depend>
```

Comment:
can we remove '<depend>autoware_planning_msgs</depend>' ?

Reply:
Thanks, remove the unnecessary dependency https://github.com/tier4/AutowareArchitectureProposal.iv/pull/698/commits/bad25fd1c825e292b812f56425005a4af05171ce